files/en-us/web/api/navigatorplugins/index.html

```diff
@@ -30,7 +30,7 @@ <h2 id="Methods">Methods</h2>
 <p><em>The <code>NavigatorPlugins</code> interface doesn't inherit any methods.</em></p>
 
 <dl>
- <dt>{{domxref("NavigatorPlugins.javaEnabled", "NavigatorPlugins.javaEnabled()")}} {{readonlyInline}}{{experimental_inline}}</dt>
+ <dt>{{domxref("NavigatorPlugins.javaEnabled", "NavigatorPlugins.javaEnabled()")}} {{experimental_inline}}</dt>
  <dd>Returns a {{domxref("Boolean")}} flag indicating whether the host browser is Java-enabled or not.</dd>
```

Comment:
```suggestion
 <dt>{{domxref("NavigatorPlugins.javaEnabled", "NavigatorPlugins.javaEnabled()")}}</dt>
 <dd>Returns false.</dd>
```
It’s also not experimental, and by spec it always returns false.
https://html.spec.whatwg.org/multipage/obsolete.html#dom-navigator-javaenabled

Reply:
Interesting, didn't know that.

Reply:
> It’s also not experimental, and by spec it always returns false.
>
>> Interesting, didn't know that.

Yeah, I vaguely recall that it’s a relatively recent spec change.